languages/js/src/filter.ts

```diff
@@ -0,0 +1,137 @@
+import type { Host } from './Host';
+import type { PolarComparisonOperator, PolarValue, obj } from './types';
+import { OsoError } from './errors';
+
+export interface SerializedRelation {
+  Relation: {
+    kind: string;
+    other_class_tag: string;
+    my_field: string;
+    other_field: string;
+  };
+}
```

Comment:
nit: This type & the corresponding `Relation::serialize` method don't feel like they're providing much value

Reply:
I replaced `Serialized*` with `obj`.